### DISCUSSION TITLE
Virtual scroll is using spacebar as a scroller

### DISCUSSION THREAD

**Discussion by CodingPhantoms** [primefaces/community#747] [PrimeNG]
Hi Team,
I have a big list of items and I am using a virtual scroll and that is where problem starts...
When virtual scroll is active, spacebar is acting as a scroller, not as a "add empty char".
I tried to prevent default and it works in order to prevent default  - prevent scroll, but still - it also prevents adding of the empty char/space.

Is this a well known bug (strange, internet is lacking this topic) or this is locally orientated problem?

**Comment by SoyDiego** [comment]
> Hi Team, I have a big list of items and I am using a virtual scroll and that is where problem starts... When virtual scroll is active, spacebar is acting as a scroller, not as a "add empty char". I tried to prevent default and it works in order to prevent default - prevent scroll, but still - it also prevents adding of the empty char/space.
> 
> Is this a well known bug (strange, internet is lacking this topic) or this is locally orientated problem?

I don't know if I understood your problem. But **maybe** if you press `spacebar` and is acting as scroller is for accessibility.
I'm not sure.

**Reply by CodingPhantoms** [reply]
Yes, you understood it correctly! :)
It seems that it is accessibility "feature", but problem is that I have <input> field inside a table, so I cannot insert empty character in that field (it will trigger scroll).

**Reply by cetincakiroglu** [reply]
Could you please share a stackblitz reproducer and guide to catch the unwanted behavior?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
